tests/inches.svx

```diff
@@ -0,0 +1,9 @@
+*units tape inches
```

Comment:
It's better to annotate the testcase with a magic comment on the first line:

```
; pos=yes warn=0
```

Then the harness knows explicitly that there should be no warnings and to check positions.

Also better to make the expected output a `.pos` rather than ` .3d` since `.pos` is human readable so makes for easier to debug test cases.  You can just convert with `survexport inches.3d inches.pos`.  That's what all the existing `cavern` testcases do - the only existing `.3d` files in the testsuite are for testing (a) `extend` and (b) support for older `.3d` file format versions.

Reply:
In fact it looks like this only currently works because it inherits the `pos` setting from whatever the previous testcase is (so it probably fails if you run just this new case with `cd tests && ./cavern.tst inches` - I haven't checked though.)